docs/DevGuide/Protocols.md

```diff
@@ -143,3 +137,17 @@ example implementation of a class that conforms to the protocol, and the
 protocol author should add it to the documentation of the protocol through a
 Doxygen snippet. This gives users a convenient way to see how the author intends
 their interface to be implemented.
+
+# Protocols and C++20 "Constraints and concepts" {#protocols_and_constraints}
+
+A feature related to protocols is proposed for C++20 and goes under the name of
```

Comment:
While squashing, this is officially in C++20 now as of last week or so. `proposed for` -> `in`